TikTokApi/tiktok.py

```diff
@@ -160,14 +169,22 @@ def getData(self, **kwargs) -> dict:
             proxies=self.__format_proxy(proxy),
         )
         try:
+            json = r.json()
+            if "type" in json and json["type"] == "verify" :
```

Comment:
```suggestion
            if json.get('type') == 'verify' :
```
A bit more concise.  `dict.get()` is one of my favourite methods in the whole python language, as it allows an EAFP approach to dictionary access operations. You almost never need to check if a key is present before accessing it with `get()`!

I also recommend using single quotes for 'token' or 'symbol' strings like these (i.e. short strings that carry meaning of their own - dictionary keys and specific values are great examples). That way you can reserve double quotes for longer strings that more for human consumption than machine. So a reader of your code can tell quickly at a glance what kind of string it is.  Another way to think about this scheme is if it would be a symbol in ruby (e.g. `:type`), then it should be single-quoted in python.

This has a knock-on benefit as well when you start using f-strings.  Because you can put nearly any python expression inside `{}` in f-strings, it won't be long until you want to pull a value out of a dict inside an f-string.  If you always use double quotes to bound an f-string, and always use single quotes for dict keys, then you can write an f-string like this every time:

```
some_string = f"Hello, my name is {protagonist['full_name']}. You {enemy['crime']}. Prepare to {enemy['punishment']."
```